css/mobileStyle.css

```diff
@@ -229,7 +229,6 @@ body {
   margin: auto;
 }
 
-
 /* ABOUT PAGE STYLES */
```

Comment:
Great design @danielochuba 🤝

- [ ] As stated in the project requirements, kindly stick to the design guidelines as much as possible. You may make sure your about footer has a dark background and kindly reduce its size.
- Your About Footer
![about-footer](https://user-images.githubusercontent.com/63917725/219565482-6a04c0f9-36b6-45b1-982c-0f8d5be55e47.PNG)
- Given About Footer
![given-about-footer](https://user-images.githubusercontent.com/63917725/219565571-364a368e-33d3-4852-9a56-77306267e9fa.PNG)

Reply:
@1GORDON thanks, but i need to know if the project is okay before making a video